### ISSUE TITLE
Retrieving the GPG key for the dpkg Yarn repository fails on Travis CI

### ISSUE THREAD

**Issue by jrgriffiniii** [pulibrary/princeton_ansible#443]
https://travis-ci.org/pulibrary/princeton_ansible/jobs/486928248 is an example of a build failure which needed to be restarted:
```
fatal: [instance]: FAILED! => {"changed": false, "cmd": "/usr/bin/apt-key adv --no-tty --keyserver hkp://pool.sks-keyservers.net --recv 72ECF46A56B4AD39C907BBB71646B01B86E50310", "msg": "Error fetching key 72ECF46A56B4AD39C907BBB71646B01B86E50310 from keyserver: hkp://pool.sks-keyservers.net", "rc": 2, "stderr": "Warning: apt-key output should not be parsed (stdout is not a terminal)\ngpg: keyserver receive failed: Cannot assign requested address\n", "stderr_lines": ["Warning: apt-key output should not be parsed (stdout is not a terminal)", "gpg: keyserver receive failed: Cannot assign requested address"], "stdout": "Executing: /tmp/apt-key-gpghome.WkjWiW5fy8/gpg.1.sh --no-tty --keyserver hkp://pool.sks-keyservers.net --recv 72ECF46A56B4AD39C907BBB71646B01B86E50310\n", "stdout_lines": ["Executing: /tmp/apt-key-gpghome.WkjWiW5fy8/gpg.1.sh --no-tty --keyserver hkp://pool.sks-keyservers.net --recv 72ECF46A56B4AD39C907BBB71646B01B86E50310"]}
```

**Comment by jrgriffiniii** [comment]
This seems to be occurring quite frequently for https://travis-ci.org/pulibrary/princeton_ansible/jobs/500014074.  I'll need to keep rebuilding this, but it might render https://github.com/pulibrary/princeton_ansible/issues/455 a higher priority if this would stabilize the build.

**Comment by kayiwa** [comment]
I think we can close this, yes? @jrgriffiniii

**Comment by jrgriffiniii** [comment]
Resolved with #625
